R/utils.R

```diff
@@ -39,9 +39,19 @@ r_app_stdx <- function(stream) {
     (is_stdout(stream) || is_stderr(stream))
 }
 
+is_rkward <- function() {
+    "rkward" %in% (.packages())
+}
+
+rkward_stdx <- function(stream) {
+  interactive() &&
+    is_rkward() &&
+    (is_stdout(stream) || is_stderr(stream))
+}
+
 is_supported <- function(stream) {
   is_option_enabled() &&
-    (isatty(stream) || r_studio_stdx(stream) || r_app_stdx(stream))
+    (isatty(stream) || r_studio_stdx(stream) || r_app_stdx(stream)) || rkward_stdx(stream)
```

Comment:
Are you sure you want to close that paren there?

Reply:
I am sure that I don't want to, sorry. The new commit should fix this. By the way, is there a reason for not simply using:
```R      
(is_stdout(stream) || is_stderr(stream)) && interactive()
```
?

Reply:
Probably could do. Just being cautious, in case some environments might not support `\r`.